### PR TITLE
Add ability to change lwfc focus graph margins

### PR DESCRIPTION
**Pull request by portante**
If one would want to place an x-axis label below both the graph and its
focus graph, prior to this patch it is not possible in a fixed height
setting, as there is no way to change the bottom margin to account for
a label.

This change adds the ability to change the bottom margin.  Note that in
order for that label to work while maintaining the focus graph height,
one also has to up the "focusHeight" value.

### CODE REVIEW

**Comment by rreinhardt9** [comment]
@portante thanks for this, I just ran into this very situation!

File: src/models/lineWithFocusChart.js [hunk 1]
```diff
@@ -494,6 +494,12 @@ nv.models.lineWithFocusChart = function() {
             margin.bottom = _.bottom !== undefined ? _.bottom : margin.bottom;
             margin.left   = _.left   !== undefined ? _.left   : margin.left;
         }},
+        margin2: {get: function(){return margin2;}, set: function(_){
```
Comment (liquidpele) [inline src/models/lineWithFocusChart.js]: I think this should probably be "focusMargin" to match the "focusHeight" option naming convention...

Comment (portante) [inline src/models/lineWithFocusChart.js]: That makes sense.  I'll make that change.

**Comment by portante** [comment]
Closing submitted a new pull request, #1080, with the update.

**Comment by portante** [comment]
Closed the wrong one.

**Comment by liquidpele** [comment]
Looks like you picked up a bunch of other commits... need to "git rebase origin/master" :)

**Comment by portante** [comment]
Okay rebased, sorry, missed that step.

**Comment by liquidpele** [comment]
oh, nm, I updated dev branch to fix that.

**Comment by liquidpele** [comment]
or maybe you fixed it then, lol.

**Comment by portante** [comment]
I just did a quick rebase and that seems to have removed that last merge commit from this pr.

Let me know if there are any other problems!

**Pull request merged**